CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 ### Master
 
+### 0.7.9
+
+* Bug fix(2): TypeError: Cannot read property 'document' of undefined
+
+### 0.7.8
+
+* Show the flow-check status using a spinner in the higher priority.
+
+### 0.7.7
+
+* Serialize the flow-server operations (including startup and ping) not to launch too many flow-server processes due to the data race.
+* Turn on more logs for the development mode.
+
+### 0.7.5
+
+* Make the variable `${workspaceRoot}` compatible with the VSCode workspace folders.
+
```

Comment:
We've not made a release, so all of those versions are still not used in this project